src/Chain.php

```diff
@@ -113,9 +113,9 @@ public function between($min, $max)
      *
      * @return $this
      */
-    public function bool()
+    public function boolean()
```

Comment:
I don't think we should rename the Chain function at all, as it's only the class that's a problem for PHP7. Check https://github.com/particle-php/Filter/blob/master/src/FilterResource.php#L63